content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -295,63 +289,139 @@ kubectl cluster-info dump
 
 ## Enabling shell autocompletion
 
-kubectl includes autocompletion support, which can save a lot of typing!
+kubectl provides autocompletion support for Bash and Zsh, which can save you a lot of typing!
 
-The completion script itself is generated by kubectl, so you typically just need to invoke it from your profile.
+Below are the procedures to set up autocompletion for Bash and Zsh. For Bash, it furthermore makes a difference whether you use Linux or macOS.
```

Comment:
i wonder if we can re-worded here, particular _For Bash, it furthermore makes a difference whether you use Linux or macOS_

maybe 

> Below are the procedures to set up autocompletion for Bash (including the difference between Linux and macOS) and Zsh. 

@zacharysarah you very good at ;) wdyt ?

Reply:
To me, your suggestion sounds good, at least 👍

Reply:
@DanyC97 💯

Reply:
Changed according to @DanyC97's suggestion.

Reply:
thanks @weibeld !